src/libraries/System.IO.Compression.ZipFile/tests/ZipFile.Unix.cs

```diff
@@ -155,7 +155,7 @@ public async Task CanZipNamedPipe()
             await Task.WhenAll(
                 Task.Run(() =>
                 {
-                    using FileStream fs = new (fifoPath, FileMode.Open, FileAccess.Write, FileShare.Read);
+                    using FileStream fs = new (fifoPath, FileMode.Open, FileAccess.Write, FileShare.Read, bufferSize: 0);
```

Comment:
This is unrelated to iOS and instead for my comments on the previous PR? Just confirming.

Reply:
> This is unrelated to iOS and instead for my comments on the previous PR? Just confirming.

Yes, exactly. I did not want to send a new PR for this small change so I've used the test failure as an opportunity to address the old feedback as well.